packages/nimble-components/src/select/testing/select.pageobject.ts

```diff
@@ -131,13 +132,14 @@ export class SelectPageObject {
     }
 
     public clickOption(index: number): void {
```

Comment:
Can you add a JSDoc comment that notes this behavior? And also mention what visible is defined to mean from the client's point of view. Not `hidden` and matches the filter? (I can't recall if `visually-hidden` is set by things other than filtering)